helm/templates/config-service-config.yaml

```diff
@@ -11,15 +11,29 @@ data:
 
     generic.config.service {
       document.store {
-        {{- $dst := .Values.configServiceConfig.dataStoreType }}
+        {{- if .Values.database }}
+        {{- $dst := .Values.database.type }}
         dataStoreType = {{ $dst }}
+        appName = {{ .Values.configServiceConfig.name }}
```

Comment:
This should be the service name and not the config name. Could you please introduce a separate section called `service` under value.yaml and add the name there to reference it?